frontend/src/components/ProductItem/ProductItem.js

```diff
@@ -1,25 +1,25 @@
 import React from 'react';
 
 import roundedPlusIcon from "../../assets/rounded_plus.svg";
-import penIcon from "../../assets/pen.svg";
 
-const ProductItem = ({ type, price, image }) => {
-  const productIcons = {
-    add: roundedPlusIcon,
-    edit: penIcon,
-    remove: penIcon,
-  }
```

Comment:
This might break the company owner view

Reply:
I know, I figured this had to be refactored anyway, but I'll add it back for now.